### PR TITLE
Add a notebook example for deploying pre-trained TorchVision model with Neo

### PR DESCRIPTION
**Pull request by hcho3**
The notebook shows an end-to-end workflow to deploy pre-trained TorchVision models with Amazon SageMaker Neo API.

### CODE REVIEW

File: sagemaker-python-sdk/pytorch_torchvision_neo/pytorch_torchvision_neo.ipynb [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    "\n",
+    "model_key = '{}/model/model.tar.gz'.format(compilation_job_name)\n",
+    "model_path = 's3://{}/{}'.format(bucket, model_key)\n",
+    "boto3.resource('s3').Bucket(bucket).upload_file('model.tar.gz', model_key)\n",
+    "\n",
+    "sm_client = boto3.client('sagemaker')\n",
+    "data_shape = '{\"input0\":[1,3,224,224]}'\n",
+    "target_device = 'ml_c5'\n",
+    "framework = 'PYTORCH'\n",
+    "framework_version = '0.4.0'\n",
```
[the remark this answers is not in the document]
Comment (hcho3) [reply sagemaker-python-sdk/pytorch_torchvision_neo/pytorch_torchvision_neo.ipynb]: PyTorch 1.0 is not yet support in Neo. We are currently working on it.

@markrogersjr Do we have ETA for general availability for 1.0 support?

[the remark this answers is not in the document]
Comment (markrogersjr) [reply sagemaker-python-sdk/pytorch_torchvision_neo/pytorch_torchvision_neo.ipynb]: We should have support for all PyTorch 1.0.x models (that can be exported with `torch.jit.trace` by the end of the week, hopefully.

**Comment by hcho3** [comment]
@nadiaya Since we are using a pre-trained model, we aren't able to use the Python SDK. Let me move the files to another directory, once we're done revising the notebook.

**Comment by hcho3** [comment]
@nadiaya I moved the notebook to advanced functionality section.

**Comment by hcho3** [comment]
@nadiaya Any updates?

**Comment by austinmw** [comment]
@hcho3 Hi, running this notebook unchanged in new notebook compilation fails for me, using pytorch_36 kernel.

**Comment by hcho3** [comment]
@austinmw I was not able to reproduced the issue. I just tried running the notebook using a fresh SageMaker notebook instance, and the model compiles fine. What kind of error message did you get?

**Pull request merged**